bigquery/cloud-client/query.py

```diff
@@ -72,10 +96,19 @@ def query_standard_sql(query):
         '--use_standard_sql',
         action='store_true',
         help='Use standard SQL syntax.')
+    parser.add_argument(
+        '--destination_table',
+        type=str,
+        help=(
+            'Destination table to use for results. '
+            'Example: my_dataset.my_table'))
 
     args = parser.parse_args()
 
     if args.use_standard_sql:
         query_standard_sql(args.query)
+    elif args.destination_table:
+        dataset, table = args.destination_table.split('.')
```

Comment:
We could use a simple regular expression to validate that the destination table name is in the correct format, i.e. dataset_name.table_name

Reply:
However, this being a sample, it might be a better idea to keep unnecessary complexity down. So I'll leave that to you

Reply:
I'd rather keep the complexity low.